README.md

```diff
@@ -70,29 +70,30 @@ jobs:
     runs-on: ubuntu-latest
     name: Update README from json data
     steps:
-    - uses: actions/checkout@v2
-    - name: Read/Write data into README
-      uses: eddiejaoude/github-actions-reader-writer@v0.1
-      with:
-        json-file-path: 'data.json'
-        github-token: ${{ secrets.GITHUB_TOKEN }}
-        columns: 3
-        object-field-names: '[ "githubUsername", "name", "imageUrl", "issueNumber" ]'
-        file-to-use: 'README.md'
-        html-cell: '<td align="center"><p><a href="https://github.com/{{ githubUsername }}">{{ name }}</a></p><img src="{{ imageUrl }}" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/{{ issueNumber }}">(:100: give your vote)</a></p></td>'
+      - uses: actions/checkout@v2
+      - name: Read/Write data into README
+        uses: eddiejaoude/github-actions-reader-writer@v0.1
+        with:
+          json-file-path: "data.json"
+          github-token: ${{ secrets.GITHUB_TOKEN }}
+          columns: 3
+          object-field-names: '[ "githubUsername", "name", "imageUrl", "issueNumber" ]'
+          file-to-use: "README.md"
+          html-cell: '<td align="center"><p><a href="https://github.com/{{ githubUsername }}">{{ name }}</a></p><img src="{{ imageUrl }}" /><p><a href="https://github.com/EddieJaoudeCommunity/awesome-github-profiles/issues/{{ issueNumber }}">(:100: give your vote)</a></p></td>'
 ```
 
 ### Json file
 
 ```typescript
 [
-    {
-        "name": "Akas Rai",
-        "githubUsername": "akasrai",
-        "imageUrl": "https://user-images.githubusercontent.com/624760/88123456-d40df580-cbc2-11ea-9add-a7fc8675b243.png",
-        "issueNumber": 12
-    }
-]
+  {
```

Comment:
The indentation here seems to be wrong. The previous one was fine, IMO.
I believe other instances should be updated to have the correct indentation as well.

Reply:
I will have to check the indentation. Is there a guide?

Reply:
> I will have to check the indentation. Is there a guide?

Not necessarily. For JSON, it's usually a 2-line indentation.

Reply:
Thank you